src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -269,7 +275,13 @@ not available, it resumes spec-defined fallabck chain from that point.
 The `scope` defines the set of URLs that the browser considers to be within your
 app, and is used to decide when the user has left the app. The `scope`
 controls the URL structure that encompasses all the entry and exit points in
-your web app. Your `start_url` must reside within the `scope`.
+your web app. For example, the next `scope` limits navigation to a subdirectory:
+
```

Comment:
adding and example would be ok. But as written here the explanation is not clear. It should say something like "with the scope below any navigation outside of `subdirectory` is considered a navigation outside of your app.